### DISCUSSION TITLE
Microphone not working in sandboxed Electron app (Webcord)

### DISCUSSION THREAD

**Discussion by Hackelhack** [sandboxie-plus/Sandboxie#1118] [Q&A Feedback] [answered]
Webcord (https://github.com/SpacingBat3/WebCord) An exceptional more private alternative to the Discord app has odd microphone issues. I'm personally aware of the bugs that electron has with microphone access - but none of the workarounds seem to fix it. 

While sandboxed, the electron app Webcord fails to get access to the microphone. You cannot unmute in the discord GUI. 
Oddly; in settings when testing the microphone; you can still hear the feed just fine. Just only when testing in the settings.

**Comment by isaak654** [comment, marked as answer]
This is more an issue report than a common discussion, so I reposted it here:  #1119
